packages/fast-components-react-base/src/utilities/extractElement.ts

```diff
@@ -0,0 +1,26 @@
+import ReactDOM from "react-dom";
+import { isNil } from "lodash-es";
+
+/**
+ * Attempts to extract an html element from a ref
+ */
+export function extractElementFromRef(
```

Comment:
I'd prefer to see a name more inline with the function implementation. This name implies it accepts a ref and returns an element, but it actually accepts a ref *or* an element. Can you either update the name to be more comprehensive or limit the scope of the function?

Reply:
Any suggestions?  I don't want to limit the scope as part of usefulness is in differentiating when a ref or element is passed in directly as we seem to be doing that in props with some regularity.

Reply:
renamed to "extractHtmlElement" in the other PR

Reply:
(if further discussion required let's do it there - https://github.com/microsoft/fast-dna/pull/2613)